package.json

```diff
@@ -126,7 +126,7 @@
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-react": "^7.21.3",
     "esm": "^3.2.25",
-    "husky": "^4.3.0",
```

Comment:
@chriswenzy why did you bump the husky package version?

Reply:
Oh no problems i have fixed the issue, am on the old version now